orchagent/txmonorch.cpp

```diff
@@ -0,0 +1,199 @@
+#include "txmonorch.h"
+#include "schema.h"
+#include "logger.h"
+#include "timer.h"
+#include "converter.h"
+#include "portsorch.h"
+
+extern PortsOrch* gPortsOrch;
+
+TxMonOrch::TxMonOrch(TableConnector confDbConnector, TableConnector stateDbConnector) :
+    Orch(confDbConnector.first, confDbConnector.second),
+    m_countersDb(COUNTERS_DB, DBConnector::DEFAULT_UNIXSOCKET, 0),
+    m_countersTable(&m_countersDb, COUNTERS_TABLE),
+    m_countersPortNameTable(&m_countersDb, COUNTERS_PORT_NAME_MAP),
+    m_portsStateTable(stateDbConnector.first, stateDbConnector.second),
+    m_pollPeriodSec(DEFAULT_POLLING_PERIOD),
+    m_thresholdPackets(DEFAULT_THRESHOLD)
+{
+    SWSS_LOG_ENTER();
+    auto interv = timespec { .tv_sec = m_pollPeriodSec, .tv_nsec = 0 };
+    m_timer = new SelectableTimer(interv);
+    auto executor = new ExecutableTimer(m_timer, this, "POLLING_TIMER");
+    Orch::addExecutor(executor);
+    m_timer->start();
+}
+
+TxMonOrch::~TxMonOrch()
+{
+    SWSS_LOG_ENTER();
+}
+
+void TxMonOrch::setTimer(uint32_t interval)
+{
+    SWSS_LOG_ENTER();
+    auto interv = timespec { .tv_sec = interval, .tv_nsec = 0 };
+
+    m_timer->setInterval(interv);
+    m_timer->reset();
+    m_pollPeriodSec = interval;
+
+    SWSS_LOG_DEBUG("m_pollPeriod set [%u]", m_pollPeriodSec);
+}
+
+void TxMonOrch::handlePeriodUpdate(const vector<FieldValueTuple>& data)
+{
+    SWSS_LOG_ENTER();
+    uint32_t periodToSet = 0;
+
+    for (auto element : data) {
```

Comment:
@liorghub 
general comment - in SONiC it is good practice to have blocks in the following way:
```
for (auto element : data)
{
    ...
}
```
can you please change it for all occurrences in code? (also refers to if-else blocks etc.)